### DISCUSSION TITLE
expose RemoteCertificateValidationCallback in ActiveMQ.Artemis.Client.Builders

### DISCUSSION THREAD

**Discussion by rsr-maersk** [Havret/dotnet-activemq-artemis-client#463] [General]
Hi, love the package! For the SSL is there any way can expose the connectionFactory.SSL.RemoteCertificateValidationCallback. We need to validate the remote cert.

**Comment by Havret** [comment]
Hi @rsr-maersk 
Not right now, but I could add it. I should have some time over the weekend, so please stay tuned.

**Reply by rsr-maersk** [reply]
Awesome. I am adding the Mass Transit now, if you don't do it for ActiveMQ.Artemis.Client then I will try after MT. OK?

**Reply by Havret** [reply]
Sure thing.

**Comment by Havret** [comment]
Hi @rsr-maersk, could you please check [2.14.0-preview5](https://www.nuget.org/packages/ArtemisNetClient/2.14.0-preview5) (https://github.com/Havret/dotnet-activemq-artemis-client/issues/465 for reference).
